rsfmodel/rsf.py

```diff
@@ -194,12 +194,30 @@ def solve(self, **kwargs):
 
 def phasePlot(system):
```

Comment:
If you're sure of the fixed size here, feel free to stick with it. But here's a pattern I often use:

``` python
def phasePlot(system, fig=None):
    if fig is None:
        fig = plt.figure()